src/api/aptos.ts

```diff
@@ -106,6 +111,7 @@ applyMixin(Aptos, Coin, "coin");
 applyMixin(Aptos, Collection, "collection");
 applyMixin(Aptos, Event, "event");
 applyMixin(Aptos, Faucet, "faucet");
+applyMixin(Aptos, FungibleAsset, "fungible_asset");
```

Comment:
Let's avoid snake case and have everything use camelcase.  I'm going to change the rest but it ends up with a lot of conflicts

Reply:
hmm wait, thats the class name, I think it make sense for snake case? Similar to `TransactionSubmission` on line 109